src/AbstractPlugin.php

```diff
@@ -207,6 +209,6 @@ private function getViewFactory(
      */
     protected function getEnvironmentSetUp($app): void
     {
-        // ..
+        $app['config']->set('app.key', 'AckfSECXIvnK5r28GVIWUAxmbBSjTsmF');
```

Comment:
We need a default app key set. Security doesn't matter and it can be exposed since it's only used for testing